libs/net/dal/Configuration/UserColleagueConfiguration.cs

```diff
@@ -12,9 +12,9 @@ public override void Configure(EntityTypeBuilder<UserColleague> builder)
         builder.Property(m => m.UserId).IsRequired().ValueGeneratedNever();
         builder.Property(m => m.ColleagueId).IsRequired().ValueGeneratedNever();
 
-        builder.HasOne(m => m.User).WithMany(m => m.UsersManyToMany).HasForeignKey(m => m.UserId).OnDelete(DeleteBehavior.Cascade);
-        builder.HasOne(m => m.Colleague).WithMany(m => m.ColleaguesManyToMany).HasForeignKey(m => m.ColleagueId).OnDelete(DeleteBehavior.Cascade);
-    
+        builder.HasOne(m => m.User).WithMany(m => m.ColleaguesManyToMany).HasForeignKey(m => m.UserId).OnDelete(DeleteBehavior.Cascade);
```

Comment:
@fbarreta I'm 99% this is the correct relationship.  If this is wrong, please let me know.

Reply:
We have a Colleague property, that is the type of User. So I think the last update was correct. A Colleague of type User has Many ...

Reply:
This is the visual of the relationship.  This is what the changes I have made should do.

![image](https://github.com/bcgov/tno/assets/3180256/a5ae333b-76e2-43b5-93bf-39d51849ceb9)